components/RichText.tsx

```diff
@@ -93,9 +93,12 @@ export const RichText = ({
     nodeResolvers: {
       heading: (children, props) => {
         const { level } = props;
-        // All heading sizes are type-1 if using the "card" type WYSIWYG
-        // For regular main content WYSIWYG, this gets you type-4 for h2, type-3 for h3, etc.
-        const headingSize = type === 'default' ? 6 - level : 1;
+        /**
+         * All heading sizes are type-1 if using the "card" type WYSIWYG
+         * For regular main content WYSIWYG, this gets you type-3 for h2, type-2 for h3, type-1 for h4
+         * but h5 and h6 would be type-0 (the minimum font size)
+         */
+        const headingSize = type === 'default' ? Math.max(5 - level, 0) : 1;
```

Comment:
Stepping down WYSIWYG heading sizes by 1 modular type. This keeps the minimum at type-0 (same size as regular paragraph)